### PR TITLE
added integration with aiohttp

### PR DESCRIPTION
**Pull request by amenezes**
Added initial integration with aiohttp #17.

@MatanRubin would you rather I use the notation below to fix the pylint warning or is there any other?
- https://github.com/SolarEdgeTech/pyctuator/runs/828838499?check_suite_focus=true#step:12:11

```python
# current code
async def get_info(request: web.Request) -> web.Response:
            return web.json_response(self._to_dict(pyctuator_impl.app_info))

# fix W0613: Unused argument 'request' (unused-argument)
async def get_info(_: web.Request) -> web.Response:
            return web.json_response(self._to_dict(pyctuator_impl.app_info))
```

### CODE REVIEW

**Comment by michaelyaakoby** [comment]
Hey Alexander, that was fast. Awesome.

While I didn't yet review this change, looking at the failed checks, I think many of them because the `aiohttp` extra module needs to be installed before pylint/mypy.
To do so, add `--extras aiohttp` to `run: poetry install --extras flask --extras fastapi --extras db --extras redis` in `.github\workflows\python_package_build.yml`.

If `aiohttp` doesn't publish stub, mypy will continue complaining, in this case you can exclude it in `mypy.ini`.

Also, pylint is complaining that `Unused argument 'request' (unused-argument)`, guess you should tell it to ignore, add `unused-argument` to `# pylint: disable=`.

Thanks

File: examples/Flask/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# FastAPI example
```
Comment (michaelyaakoby) [inline examples/Flask/README.md:1]: Oops... guess it was my bad :-)

File: pyctuator/impl/aiohttp_pyctuator.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+from multidict import CIMultiDict
+
+from pyctuator.impl.pyctuator_impl import PyctuatorImpl
+from pyctuator.impl.pyctuator_router import PyctuatorRouter
+from pyctuator.httptrace import TraceRecord, TraceRequest, TraceResponse
+from pyctuator.impl import SBA_V2_CONTENT_TYPE
+
+
+# pylint: disable=too-many-locals
+# pylint: disable=unused-argument
```
Comment (michaelyaakoby) [inline pyctuator/impl/aiohttp_pyctuator.py:19]: You can put the two "disables" in one line with commas between them.
Not critical to me

**Comment by amenezes** [comment]
Thank you for your help @michaelyaakoby. The commented steps solved the issue relative to CI.

The code base of the project is cohesing and I would really like to use this feature in an application in my work.

**Comment by michaelyaakoby** [comment]
Hi @amenezes , I've created a pull-request to your repo adding aiohttp to E2E test.
Please review https://github.com/amenezes/pyctuator/pull/1
Thanks.

**Comment by amenezes** [comment]
Sorry for delay @michaelyaakoby I made merge with your PR but now some error is happening with the aiohttp tests.

**Comment by michaelyaakoby** [comment]
@amenezes, the failures seem like issues in `pyctuator/impl/aiohttp_pyctuator.py`, will be great if you can try and fix this (run `tests.test_pyctuator_e2e.py` specifying 'aiohttp' keyword).
I'll be able to look at this earlier next week.
Thanks

**Comment by michaelyaakoby** [comment]
Hi @amenezes I'm merging this to a branch where I'll add the E2E fix (which I sent you earlier this week) and we should release shortly.
Thanks
Michael

**Pull request merged**